# Probability

## Reto 1
Hay una caja con 10 bolas blancas, 12 bolas rojas y 8 bolas negras. Calcula la probabilidad de:
* Sacar una bola blanca.
* Sacar una bola blanca en la primera extracción y una bola negra en la segunda extracción sin reemplazar la primera.
* Sacar una bola blanca en la primera extracción y una bola negra en la segunda extracción reemplazando la primera.
* Sacar 3 bolas y que sean todas de diferentes colores.

- Una bola blanca

\begin{equation*}
\begin{aligned}
P(\text{una bola blanca}) &= \frac{10}{10+12+8} = \frac{10}{30} = \frac{1}{3} \\
\end{aligned}
\end{equation*}

- Sacar una bola blanca en la primera extracción y una bola negra en la segunda extracción sin reemplazar la primera.

\begin{equation*}
\begin{aligned}
P(\text{blanca primera extracción}) &= \frac{1}{3} \\
P(\text{negra segunda extracción reemplazando}) &= \frac{8}{29} \\
P(\text{combinada}) &= \frac{1}{3} \times \frac{8}{29} = \frac{8}{87} \\
\end{aligned}
\end{equation*}

- Sacar una bola blanca en la primera extracción y una bola negra en la segunda extracción reemplazando la primera.

\begin{equation*}
\begin{aligned}
P(\text{blanca primera extracción}) &= \frac{1}{3} \\
P(\text{negra segunda extracción reemplazando}) &= \frac{8}{30} \\
P(\text{combinada}) &= \frac{1}{3} \times \frac{8}{30} = \frac{8}{90} \\
\end{aligned}
\end{equation*}

- Sacar 3 bolas y que sean todas de diferentes colores.

Hay 6 posibilidades pero todas son simétricas entre ellas.  
Se comprueban dos de ellas para ver si esto es cierto:

\begin{equation*}
\begin{aligned}
P(\text{blanca primera extracción}) &= \frac{1}{3} \\
P(\text{negra segunda extracción}) &= \frac{8}{29} \\
P(\text{roja tercera extracción}) &= \frac{12}{28} = \frac{3}{7} \\
P(\text{combinación 3 sucesos}) &= \frac{1}{3} \times \frac{8}{29} \times \frac{3}{7} = \frac{24}{609} = \frac{8}{203} \\
\end{aligned}
\end{equation*}

\begin{equation*}
\begin{aligned}
P(\text{roja primera extracción}) &= \frac{12}{30} \\
P(\text{blanca segunda extracción}) &= \frac{10}{29} \\
P(\text{negra tercera extracción}) &= \frac{8}{28} \\
P(\text{combinación 3 sucesos}) &= \frac{12}{30} \times \frac{10}{29} \times \frac{8}{28} = \frac{960}{24360} = \frac{8}{203} \\
\end{aligned}
\end{equation*}

Entonces multiplicamos esta probabilidad entre 6 y nos da la probabilidad total de sacar 3 bolas diferentes seguidas.

\begin{equation*}
P(6 \text{ variantes}) = \frac{8}{203} \times 6 = \frac{48}{203}
\end{equation*}


## Reto 2

Quieres ir de picnic el sábado. Odias la lluvia y quieres planificar tu fin de semana. Para ayudarte a tomar una decisión, recopilas los siguientes datos sobre los días lluviosos:

* El 40% de todos los días están nublados.
* Aproximadamente el 50% de los días nublados son lluviosos.
* Aproximadamente el 20% de los días no nublados son lluviosos.

¿Cuál es la probabilidad de que llueva el próximo sábado?

In [2]:
# Se combinarían la posibilades de 40% de dias * 50% de esos días mas el 60% por 20% de los no nublados
p = (0.4*0.5)+(0.6*0.2)
p

0.32

Por lo tanto la probabilidad es de lluvia es de 
0.32 * 100 = 32%

## Reto 3

A y B son eventos de una probabilidad tal que \(P(A) = 0.3\), \(P(B) = 0.6\) y \(P(A \cap B) = 0.1\)

¿Cuáles de las siguientes afirmaciones son verdaderas?
**SUGERENCIA: dibuja un gran diagrama de Venn a tu lado**
* \(P($\overline{A}$) = 0.7\)
* \(P($A$ $\cup$ B) = 0.6\)
* \(P($A$ $\cap$ $\overline{B}$) = 0.2\)
* \(P($A$ $\cup$ $\overline{A}$) = 1\)  
* \(P($A$ $\cap$ ($B$ $\cup$ $\overline{B}$)) = 0.4\)
* \(P($\overline{A}$ $\cap$ $\overline{B}$) = 0.3\)
* \($P$($\overline{A}$ $\cap$ $B$) = 0.9\)  

\begin{align*}
1. & \quad P(\overline{A}) = 1 - P(A) = 1 - 0.3 = 0.7 \quad \text{(Verdadera)} \\
2. & \quad P(A \cup B) = P(A) + P(B) - P(A \cap B) = 0.3 + 0.6 - 0.1 = 0.8 \quad \text{(Falsa)} \\
3. & \quad P(A \cap \overline{B}) = P(A) - P(A \cap B) = 0.3 - 0.1 = 0.2 \quad \text{(Verdadera)} \\
4. & \quad P(A \cup \overline{A}) = P(\Omega) = 1 \quad \text{(Por definición el espacio muestral completo es 1. Verdadera)} \\
5. & \quad P(A \cap (B \cup \overline{B})) = P(A \cap \Omega) = P(A) = 0.3 \quad \text{(Verdadera)} \\
6. & \quad P(\overline{A} \cap \overline{B}) = 1 - P(A \cup B) = 1 - 0.8 = 0.2 \quad \text{(Falsa)} \\
7. & \quad P(\overline{A} \cap B) = P(B) - P(A \cap B) = 0.6 - 0.1 = 0.5 \quad \text{(Verdadera)} \\
\end{align*}

## Reto 4

* Si alguien viaja a China, la probabilidad de contraer Covid es del 1%
* Si alguien viaja a Italia, la probabilidad de contraer Covid es del 20%

Lanzo un dado con números 1, 2, 3, 4, 5, 6:
* Si sale 1: Vuelo a Italia
* Si no sale 1: Vuelo a China

Durante mi viaje, contraje Covid. ¿Cuál es la probabilidad de que haya volado a China?

\begin{align*}
I & : \text{Volé a Italia} \\
C & : \text{Volé a China} \\
K & : \text{Contraje Covid}
\end{align*}

El dado tiene 6 caras:
\begin{align*}
P(I) &= \frac{1}{6} \\
P(C) &= \frac{5}{6}
\end{align*}

Las probabilidades de contraer Covid en cada país son:
\begin{align*}
P(K|I) &= 0.20 \\
P(K|C) &= 0.01
\end{align*}

Queremos encontrar \(P(C|K)\), la probabilidad de haber volado a China dado que contrajo Covid. Usamos el teorema de Bayes:

\begin{align*}
P(C|K) &= \frac{P(K|C) \cdot P(C)}{P(K)} \\
P(K) &= P(K|I) \cdot P(I) + P(K|C) \cdot P(C) \\
P(K) &= (0.20 \cdot \frac{1}{6}) + (0.01 \cdot \frac{5}{6}) \\
P(K) &= \frac{0.20}{6} + \frac{0.05}{6} \\
P(K) &= \frac{0.25}{6} \\
P(K) &= \frac{1}{24}
\end{align*}

Entonces, ya con todas las variables necesarias:

\begin{align*}
P(C|K) &= \frac{P(K|C) \cdot P(C)}{P(K)} \\
P(C|K) &= \frac{0.01 \cdot \frac{5}{6}}{\frac{1}{24}} \\
P(C|K) &= \frac{0.01 \cdot 5 \cdot 24}{6} \\
P(C|K) &= \frac{0.01 \cdot 120}{6} \\
P(C|K) &= \frac{1.2}{6} \\
P(C|K) &= 0.20
\end{align*}

Por lo tanto, la probabilidad de que haya volado a China dado que contrajo Covid es \(P(C|K) = 0.20\). Es decir, 20%.

## BONUS

Lanzas 2 dados. ¿Cuál es la probabilidad de que la suma de los números sea 7? ¿Y 8? ¿Y 9? Luego simula 10000 veces y dibuja un gráfico de conteo.

tu explicación aquí